libraries/js/immutable/list.effekt

```diff
@@ -20,6 +20,14 @@ def foreach[A](l: List[A]) { f: A => Unit } : Unit = {
   }
 }
 
+def foreachIndex[T](list: List[T], i: Int){ f: (Int, T) => Unit}: Unit = {
```

Comment:
Make a local loop (tail-called function) so users don't have to pass in the initial index

Reply:
done